custom/_legacy/mvp_docs/pillows.py

```diff
@@ -236,7 +237,7 @@ def _get_mvp_indicator_pillow(pillow_id, processor, topic):
     return ConstructedPillow(
         name=pillow_id,
         checkpoint=checkpoint,
-        change_feed=KafkaChangeFeed(topics=[topic], group_id='{}-group'.format(pillow_id)),
+        change_feed=CouchChangeFeed(XFormInstance.get_db(), include_docs=False),
```

Comment:
I was just going to leave it on couch but could be persuaded to migrate to kafka if it means there's more cleanup we can do.